src/catalogue/items/catalogueItemsDialog.component.test.tsx

```diff
@@ -865,14 +975,28 @@ describe('Catalogue Items Dialog', () => {
       createView();
 
       const nameInput = screen.getByLabelText('Name *');
-      await user.type(nameInput, newName);
+      fireEvent.change(nameInput, { target: { value: newName } });
```

Comment:
Why did this change?

Reply:
Sometimes userEvent doesn't work as expected.  Sometimes the onChangeValue changes one character at a time (instead just of "test" it would be called with "t","e","s" and "t") so you can't do the assertion check. For now I just change it to fireEvent because that works but I plan on investigated this in the future.